apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -132,7 +132,7 @@ const SwapForm = () => {
     onNoRates: trackNoRates,
     ...locationState,
     providers: storedProviders,
-    includeDEX: showDexQuotes,
+    includeDEX: showDexQuotes?.enabled | false,
```

Comment:
Should this not use || instead?

```suggestion
    includeDEX: showDexQuotes?.enabled || false,
```